tests/src/Functional/CorporateFooterRenderTest.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Drupal\Tests\oe_corporate_blocks\Functional;
+
+use Drupal\Tests\BrowserTestBase;
+use Symfony\Component\DomCrawler\Crawler;
+
+/**
+ * Test footer block rendering.
+ */
+class CorporateFooterRenderTest extends BrowserTestBase {
```

Comment:
We are just testing the accessibility link here, so either we rename the full class or the methods.